enhancements/ingress/optional-ingress.md

```diff
@@ -0,0 +1,225 @@
+---
+title: optional-ingress
+authors:
+  - "@alebedev87"
+reviewers:
+  - "@Miciah"
+  - "@bparees"
+  - "@cewong"
+  - "@wking"
+approvers:
+  - "@Miciah"
+  - "@bparees"
+  - "@wking"
+api-approvers:
+  - N/A
+creation-date: 2023-06-02
+last-updated: 2023-06-02
+tracking-link:
+  - https://issues.redhat.com/browse/NE-1129
+see-also:
+  - "/enhancements/installer/component-selection.md"
+  - "/enhancements/ingress/aws-load-balancer-operator.md"
+replaces:
+superseded-by:
+---
+
+# Make Ingress Operator optional
+
+## Summary
+
+This enhancement describes the implementation of a new capability to enable/disable the default ingress of the OpenShift cluster.
+The new capability (from here on called the ingress capability) includes [the cluster ingress operator](https://github.com/openshift/cluster-ingress-operator) as well as the custom resources managed by the operator.
+This enhancement focuses on the requirements for [RFE-3395](https://issues.redhat.com/browse/RFE-3395) which targets ROSA, a managed OpenShift service based on HyperShift. The implementation of the ingress capability on the standalone OpenShift is out of the scope of this enhancement.
```

Comment:
is this EP about users of ROSA clusters in general(which can also be traditional OCP clusters, just managed by SD), or users of Hypershift guest clusters in ROSA specifically?  I think you intend it to be only the latter.

Reply:
Yes, it was intended for the HyperShift guest clusters. I slightly changed it to better detail this.